### ISSUE TITLE
YouTube video keep playing in background.

### ISSUE THREAD

**Issue by AmitPrajapati1902** [PierfrancescoSoffritti/android-youtube-player#200]
Hello 

How to stop keep playing video in background. We have already used                 activity.lifecycle.addObserver(youTubeView) lifecycle. 

Video is not initialised yet but user is going in background that time video is keep playing in background

**Comment by PierfrancescoSoffritti** [comment]
You can solve this issue by checking the state of the lifecycle before playing the video. You can find an [example here](https://github.com/PierfrancescoSoffritti/android-youtube-player/blob/master/core-sample-app/src/main/java/com/pierfrancescosoffritti/aytplayersample/examples/basicExample/BasicExampleActivity.java#L109).

**Comment by AmitPrajapati1902** [comment]
Thank you for your quick reply.

That issue is partially resolved. After initialised it is started loading, In that scenario we are going in background that time video is playing.

**Comment by PierfrancescoSoffritti** [comment]
Sorry, I couldn't understand. Could you rephrase?

**Comment by PierfrancescoSoffritti** [comment]
As I said yesterday: to solve your issue do what I'm doing in the sample app at the url I gave you.
